contrib/electrumpersonalserver.service

```diff
@@ -0,0 +1,19 @@
+# Electrum Personal Server: systemd unit
+# /etc/systemd/system/electrumpersonalserver.service
+
+[Unit]
+Description=Electrum Personal Server
+After=bitcoind.service
+
+[Service]
+ExecStart=/usr/bin/python3 /your-installation-path/server.py  [optional arguments]
+User=bitcoin
+Group=bitcoin
+Type=simple
+Restart=on-failure
+RestartSec=60
+KillMode=process
+TimeoutSec=60
```

Comment:
I don't think you want this.  It sets a hard delay on start and stop.  While it's not an issue with stopping, for start it could be.  If startup takes longer (e.g. after you add a new address), the unit could be considered as failed.  I found the `ExecStartPre` trick more robust.

Reply:
Correction: it's actually not a delay, but more like startup duration.  I hope that makes my point clearer.

Reply:
Agreed, will remove `TimeoutSec`